wpilibc/src/main/native/cpp/MatchSpecificDataSender.cpp

```diff
@@ -0,0 +1,32 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "MatchSpecificDataSender.h"
+
+#include <networktables/NetworkTable.h>
+#include <networktables/NetworkTableEntry.h>
+#include <networktables/NetworkTableInstance.h>
+
+namespace frc {
+
+MatchSpecificDataSender::MatchSpecificDataSender() {
+  m_table = nt::NetworkTableInstance::GetDefault().GetTable("FMSInfo");
+  m_metaDataKey = m_table->GetEntry(".type");
+  m_metaDataKey.ForceSetString("2018FMSInfo");
+  m_plateConfiguration = m_table->GetEntry("PlateConfiguration");
+  m_currentData = "LLL";
```

Comment:
Is this really what we want?  Any code reading this string won't know if the LLL is real or set by the field.  Wouldn't it be better to be an empty string instead?